app/models/user.rb

```diff
@@ -105,12 +107,24 @@ def session_active?(id)
     session_activations.active? id
   end
 
+  def ldap_user?
+    Rails.configuration.x.use_ldap && ldap_dn.present?
```

Comment:
yes, sorry, I was misreading. I was confused because of the attribute :ldap_username, above.